docs/appendices/release-notes/5.7.0.rst

```diff
@@ -45,7 +45,8 @@ Version 5.7.0 - Unreleased
 Breaking Changes
 ================
 
-None
+- Added ``CURRENT_ROLE`` as a reserved keyword. If you have a column or table
+  named ``CURRENT_ROLE``, you must escape it using double quotes.
```

Comment:
Could we make this non reserved instead? As long as it doesn't cause ambiguity in the parser there isn't really a downside to it?

Reply:
Hm right, will change, thx.

Reply:
Have you seen this: https://github.com/crate/crate/pull/15560#pullrequestreview-1882566899 ?
Is there a benefit of having it non-reserved?
If we do so, we need to fix the logic where we list the reserved keywords (actually how we mark the reserved ones) as the keywords like `CURRENT_ROLE`, `CURRENT_TIME`, etc. are functions and not qualified names, (see my comment above).

Reply:
> Is there a benefit of having it non-reserved?

It wouldn't cause a breaking change.

But that these are functions is a good point. What would happen if they're non-reserved and a user tries to use a `current_role` column name?